### PR TITLE
feat: [Linux] Add FrameBuffer and libinput support

### LINKED ISSUE #5847: Framebuffer support on GTK Linux
## What would you like to be added:
Support for GTK with frame buffer.

## Why is this needed:
There are a lot of ARM devices for payment in transportation (buses, trains, etc) using NFC and RFID cards, unfortunately, there's no good framework for developing apps for these devices. Some of them are Android or Linux. 
In the case of Linux, the toolkits are quite limited to Qt applications, this is due to the fact that those devices don't use X Window interface to display the content, they are pretty simple in terms of technology compared to a Raspberry Pi, the only way is through frame buffer. As far as I know, this feature is included in the GTK library.

## For which Platform:

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Skia
  - [ ] WPF
  - [X] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

## Anything else we need to know?

It would be awesome to support this!
I have taken a look at most of the well-known cross-platform frameworks based on C# and .NET, and none of them have support for frame buffer. I found Avalonia has support but doesn't include Android.

If this feature is considered will be really appreciated! :)

### PR DESCRIPTION
**Pull request by jeromelaban**
GitHub Issue (If applicable): fixes #5847

## PR Type

What kind of change does this PR introduce?
- Feature

## What is the new behavior?

Adds support for Linux FrameBuffer and input through `libinput`. This enables Uno Platform support for systems without X11/Wayland servers.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI.Runtime.Skia.Linux.FrameBuffer/CoreWindowExtension.Mouse.cs [hunk 1]
```diff
@@ -0,0 +1,127 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```
Comment (davidjohnoliver) [inline src/Uno.UI.Runtime.Skia.Linux.FrameBuffer/CoreWindowExtension.Mouse.cs:3]: What's the source of this code? Might be useful to include it as a comment. Likewise with other files

File: src/Uno.UI.Runtime.Skia.Linux.FrameBuffer/DisplayInformationExtension.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+			=> (uint)(Renderer?.PixelSize.Height ?? 0);
+
+		public uint ScreenWidthInRawPixels
+			=> (uint)(Renderer?.PixelSize.Width ?? 0);
+
+		public float LogicalDpi
+		{
+			get
+			{
+				return 96.0f;
```
Comment (davidjohnoliver) [inline src/Uno.UI.Runtime.Skia.Linux.FrameBuffer/DisplayInformationExtension.cs:28]: will this apply to every device?

**Review by davidjohnoliver — APPROVED** [review]

File: src/Uno.UI.Runtime.Skia.Linux.FrameBuffer/CoreWindowExtension.Mouse.cs (hunk 1 above)
Comment (jeromelaban) [reply src/Uno.UI.Runtime.Skia.Linux.FrameBuffer/CoreWindowExtension.Mouse.cs:3]: Missed this, yes, I'll add it.

File: src/Uno.UI.Runtime.Skia.Linux.FrameBuffer/DisplayInformationExtension.cs (hunk 2 above)
Comment (jeromelaban) [reply src/Uno.UI.Runtime.Skia.Linux.FrameBuffer/DisplayInformationExtension.cs:28]: Yes, there's no scaling support yet (the device cannot tell).

**Review by carldebilly — APPROVED** [review]

**Review by ajpinedam — APPROVED** [review]

**Pull request merged**